posts/2019/06/python-library-method-usage.md

```diff
@@ -0,0 +1,217 @@
+<!--
+.. title: Method usage for popular numberical and scientific libraries
```

Comment:
typo "numberical"

Reply:
perhaps "Measuring function usage ..."? (IIRC you could do functions not methods?)